### PR TITLE
Replace colors by chalk

### PR DESCRIPTION
**Pull request by facuspagnuolo**
We were using [`colors`](https://www.npmjs.com/package/colors) here and [`chalk`](https://www.npmjs.com/package/chalk) in `zos-cli`. We may want to use the same dependency in both projects. this is just a proposal. For instance, it will mean one less dependency on the cli side. Thoughts?

### CODE REVIEW

**Comment by frangio** [comment]
+1 for consistency.
+1 for `chalk`.

File: src/utils/Logger.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
-import colors from 'colors'
+import chalk from 'chalk'
 
 const defaults = {
   verbose: false,
-  silent: true
+  silent: false
```
Comment (frangio) [inline src/utils/Logger.js]: Is this related to the switch to `chalk`? Should be a different PR IMO so that we can discuss it separately.

File: src/utils/Logger.js [hunk 2]
```diff
@@ -35,14 +35,11 @@ export default class Logger {
     if (this.opts.silent) {
       return;
     }
-    if (this.opts.verbose) {
-      console.error(`[${this.prefix}] ${msg}`[color])
-    } else {
-      console.error(msg[color])
-    }
+    if (this.opts.verbose) msg = `[${this.prefix}] ${msg}`
+    console.error(chalk.keyword(color).dim(msg))
```
Comment (frangio) [inline src/utils/Logger.js]: What does `dim` do? I can't find it in the documentation for `chalk`.

File: src/utils/Logger.js [hunk 3]
```diff
@@ -20,7 +20,7 @@ export default class Logger {
   }
 
   info(msg) {
-    this.log(msg, 'green')
+    this.log(msg, 'red')
```
Comment (frangio) [inline src/utils/Logger.js]: :thinking: `info` type logs sound like they shouldn't have any color, specially not red! The `log` function doesn't support no color at the moment.

**Comment by facuspagnuolo** [comment]
I pushed some testing stuff @frangio, that's why PR is needed 😅
